infra/Pos/Communication/Relay/Logic.hs

```diff
@@ -310,10 +341,11 @@ invDataListener enqueue InvReqDataParams{..} = listenerConv $ \__ourVerInfo node
             whenJust inv' $ expectInv $ \InvMsg{..} -> do
                 useful <- handleInvDo (handleInv nodeId) imKey
                 whenJust useful $ \ne -> do
```

Comment:
Sure, it could be "don't send me the data", or an error. Shall we improve this to make the distinction? Wallets will need it I guess, so they can retry unless there's an explicit "go away".